contrib/storage-http/src/main/java/org/apache/drill/exec/store/http/udfs/HttpHelperFunctions.java

```diff
@@ -153,10 +149,15 @@ public void setup() {
         drillbitContext,
         pluginName
       );
+
       endpointConfig = org.apache.drill.exec.store.http.util.SimpleHttp.getEndpointConfig(
         endpointName,
         plugin.getConfig()
       );
+
+      // Add JSON configuration from Storage plugin, if present.
+      jsonLoaderBuilder = org.apache.drill.exec.store.http.udfs.HttpUdfUtils.setupJsonBuilder(endpointConfig, loader, options);
```

Comment:
It's different here because in teh `http_response`, the configs are being pulled from a Drill storage plugin.  Whereas the `http_get` function just does a basic http GET request.

Reply:
There was a bunch of logic that was getting difficult to do with all the full paths, and it was really hard to debug.

Reply:
> There was a bunch of logic that was getting difficult to do with all the full paths, and it was really hard to debug.
Do you mean full path of classes (without imports)?

Reply:
Yes.  Plus also if the logic is not in the UDF it is easier to add breakpoints.

Reply:
It can be easly debug after uncommenting this [saveCode](https://github.com/apache/drill/blob/570ce63088ab4e2b479b53a9b41696adbb0c9278/exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/project/ProjectRecordBatch.java#L303) line of code in `ProjectRecordBatch`.
You can find more info in [GeneratedCode.md](https://github.com/apache/drill/blob/dc8d0107cb6a4a31e1beaa9c98d6e51cf7486a67/docs/dev/GeneratedCode.md#instructions-for-intellij) doc